### PR TITLE
Update for menu, sync menutype and client_id

### PR DESCRIPTION
**Pull request by rdeutz**
### Summary of Changes
We use different menutype values for the admin menu. This PR syncs the used name for updating old installations. It also sets the client_id for admin menus to 1

### Testing Instructions
* Install Joomla
* Install an extension

Make sure everything works. 

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/components/com_admin/sql/updates/postgresql/3.7.0-2017-01-17.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- Sync menutype for admin menu and set client_id correct
+UPDATE "#__menu" SET "client_id" = 1 WHERE "menutype" = 'main' OR "menutype" = 'menu';
+UPDATE "#__menu" SET "menutype" = 'main' WHERE "menutype" = 'main' OR "menutype" = 'menu';
```
Comment (zero-24) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2017-01-17.sql]: do we need to check on correct menutypes too?

Comment (rdeutz) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2017-01-17.sql]: Isn't that what we are doing here? Don't get the question.

Comment (zero-24) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2017-01-17.sql]: We check main and menu. and check everything to main.

so why not just check for menu and set them to main? As the others are correct.

Comment (rdeutz) [inline administrator/components/com_admin/sql/updates/postgresql/3.7.0-2017-01-17.sql]: just copied, it doesn't matter

**Comment by izharaazmi** [comment]
@rdeutz Do you intend to make `"menutype" = 'menu'` obsolete/deprecate?

I also find the client_id to be already "1" as of J3.6.5. Why do we need to do that update sql?

**Comment by rdeutz** [comment]
I don't see a reason to have menutype == menu, the menutype isn't used at all before 3.7. I think  I have found in old sites that has client_id = 0 for admin menu item but while thinking about it again I might be wrong. Whatever it doesn't hurt :-)

**Comment by izharaazmi** [comment]
So now should we get rid of the dropdown option "Menu (protected)" in the menu manager filter or keep it for just in case B/C?

**Comment by wojsmol** [comment]
@rdeutz This PR may cause the menu to disappear from the frontend on web pages created in older Joomla! versions and updated to 3.7.0.

**Comment by brianteeman** [comment]
@wojsmol please create a new issue and reference this one. Your comment wont be seen otherwise

**Comment by wojsmol** [comment]
@brianteeman I work on PR, give me about an hour.

**Comment by wojsmol** [comment]
@brianteeman issue created https://github.com/joomla/joomla-cms/issues/15671

**Comment by PhilETaylor** [comment]
Yes, that change is pathetic and should never have been made :-(

**Pull request merged**